### PR TITLE
[WIP] Add INT8 fuse+relu6 support and mobilenetv2 INT8 test based on PR17130

### PR DESCRIPTION
**Pull request by lidanqing-vv**
This PR failed on CI, I am working on it. 
1. add INT8 conv+ relu6 fusion support
2. add mobilenetv2 test.

After the fuse of conv+relu6, the performance is improved as follows:

INT8 fuse| Top1 Accuracy | Top5 Accuracy | Performance
| -- |-- | -- | --
Before fuse| 68.32% | 88.16% | 144.86
After fuse|71.15% |90.01% | 1193.01

INT8/FP32| Accuracy(INT8-FP32) | Performance(INT8/FP32)
| -- |-- | --
--------| -0.5%   (<1%) | 1.71

machine: Intel(R) Core(TM) i9-7940X CPU @ 3.10GHz, 14 Cores
paddle_num_threads 14

### CODE REVIEW

**Comment by luotao1** [comment]
1. please remove the unused commit since #17130 merged.

2. > the performance is improved as follows:

What's machine and how many threads do you use?

3. please see https://github.com/PaddlePaddle/Paddle/pull/17130#discussion_r286295610

File: paddle/fluid/inference/tests/api/CMakeLists.txt [hunk 1]
```diff
@@ -172,8 +172,18 @@ if(WITH_MKLDNN)
   if (NOT EXISTS ${INT8_MOBILENET_MODEL_DIR})
     inference_download_and_uncompress(${INT8_MOBILENET_MODEL_DIR} "${INFERENCE_URL}/int8" "mobilenetv1_int8_model.tar.gz" )
   endif()
+
+
+
```
Comment (wojtuss) [inline paddle/fluid/inference/tests/api/CMakeLists.txt:177]: Please, remove multiple empty lines

File: paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc [hunk 2]
```diff
@@ -377,7 +378,7 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
     key.reserve(MaxKeyLength);
     platform::ConvMKLDNNHandler::AppendKey(
         &key, src_tz, weights_tz, strides, paddings, dilations, groups, src_dt,
-        input->format(), fuse_relu, fuse_residual_conn, false /*fuse_brelu*/,
+        input->format(), fuse_relu || fuse_brelu, fuse_residual_conn, false,
```
Comment (wojtuss) [inline paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc:381]: Please, change `false` into `fuse_brelu`, the same also in the calls below.

File: paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc [hunk 3]
```diff
@@ -463,16 +464,14 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
 
         conv_pd = ConvFwdPrimitiveDesc(
             src_md, weights_md, bias_md, dst_md, strides, paddings,
-            mkldnn_engine, fuse_relu, fuse_residual_conn, false /*fuse_brelu*/,
-            0.0 /*fuse_brelu_threshold*/, output_shift_scale, sum_scale,
-            is_test);
+            mkldnn_engine, fuse_relu || fuse_brelu, fuse_residual_conn, false,
+            0.0, output_shift_scale, sum_scale, is_test);
```
Comment (wojtuss) [inline paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc:468]: ditto

File: paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc [hunk 4]
```diff
@@ -463,16 +464,14 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
...
       } else {
-        conv_pd = ConvFwdPrimitiveDesc(src_md, weights_md, dst_md, strides,
-                                       paddings, mkldnn_engine, fuse_relu,
-                                       fuse_residual_conn, false /*fuse_brelu*/,
-                                       0.0 /*fuse_brelu_threshold*/,
-                                       output_shift_scale, sum_scale, is_test);
+        conv_pd = ConvFwdPrimitiveDesc(
+            src_md, weights_md, dst_md, strides, paddings, mkldnn_engine,
+            fuse_relu || fuse_brelu, fuse_residual_conn, false, 0.0,
+            output_shift_scale, sum_scale, is_test);
```
Comment (wojtuss) [inline paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc:474]: ditto

File: paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc [hunk 5]
```diff
@@ -356,10 +356,11 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
     mkldnn::memory::data_type src_dt =
         paddle::framework::ToMKLDNNDataType(input->type());
 
-    auto dst_dt = (fuse_relu) ? paddle::framework::ToMKLDNNDataType(
-                                    framework::DataTypeTrait<uint8_t>::DataType)
-                              : paddle::framework::ToMKLDNNDataType(
-                                    framework::DataTypeTrait<int8_t>::DataType);
+    auto dst_dt = (fuse_relu || fuse_brelu)
```
Comment (wojtuss) [inline paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc:359]: The `PADDLE_ENFORCE` check for `fuse_brelu` in the line 343 can be removed now.

File: paddle/fluid/inference/tests/api/CMakeLists.txt (hunk 1 above)
Comment (lidanqing-vv) [reply paddle/fluid/inference/tests/api/CMakeLists.txt:177]: Done

**Pull request closed without merging**